kotlinx-coroutines-core/common/src/CoroutineExceptionHandler.kt

```diff
@@ -52,11 +52,37 @@ public inline fun CoroutineExceptionHandler(crossinline handler: (CoroutineConte
     }
 
 /**
- * An optional element in the coroutine context to handle uncaught exceptions.
+ * An optional element in the coroutine context to handle **uncaught** exceptions.
  *
- * Normally, uncaught exceptions can only result from coroutines created using the [launch][CoroutineScope.launch] builder.
+ * Normally, uncaught exceptions can only result from _root_ coroutines created using the [launch][CoroutineScope.launch] builder.
+ * All _children_ coroutines (coroutines created in the context of another [Job]) delegate handling of their
```

Comment:
excepts ones whose parent is a supervisor job. I think it is a good idea to mention here, Kdoc will be read much more frequently than our guide